### DISCUSSION TITLE
Energy dashboard error - Shelly off/offline for a while

### DISCUSSION THREAD

**Discussion by Tesch1933** [home-assistant/frontend#22980] [Dashboard]
Hello everyone,

I have the following problem for which I simply cannot find a solution.

My Shelly Plus 1PM records the yield of my balcony power plant (BKW).

Due to renovation work, I had to take BKW off the for a week. This meant that the Shelly was also off the grid for a week.

Now I have set everything up again and connected it. Everything works perfectly BUT.

My energy dashboard has "crashed"

I assume that this is because the Shelly hasn't sent any data to Home Assistant for a while.

Have a look at the corresponding sensor data.
Basically, everything is plausible, but Home Assistant seems to have a problem with the fact that no data was available for a while.

I hope I can find an expert here who can help me solve the problem.

Many thanks and best regards!
![IMG_6312](https://github.com/user-attachments/assets/be89d09a-f8dd-4cf7-ac6e-7a2d0f2c39dd)
![IMG_6311](https://github.com/user-attachments/assets/c018d4d5-6477-47a0-845f-3ba197d87026)

**Comment by karwosts** [comment]
I would recommend you to ask on the community forum or the discord chat server for support, this isn't typically an area for asking for help with your installation.
